spec/tbd_tests_spec.rb

```diff
@@ -2856,6 +2861,13 @@
     expect(model).to_not be_empty
     model = model.get
 
+    # As a side test, switch glass doors to (opaque) doors.
+    model.getSubSurfaces.each do |sub|
+      next unless sub.subSurfaceType.downcase == "glassdoor"
+
+      expect(sub.setSubSurfaceType("Door")).to be true
+    end
+
```

Comment:
TBD will successfully tag a balcony/subsurface edge as either **:balconysill** or **:balconydoorsill**, depending on whether the door is considered "vertically fenestrated" or not (90.1 2022 distinction).